src/zone/state/v1.rs

```diff
@@ -202,14 +208,13 @@ impl SignerPolicySpec {
 //----------- SignerSerialPolicySpec -------------------------------------------
 
 /// Policy for generating serial numbers.
-#[derive(Copy, Clone, Debug, Default, Deserialize, Serialize)]
+#[derive(Copy, Clone, Debug, Deserialize, Serialize)]
 #[serde(rename_all = "kebab-case", deny_unknown_fields)]
 pub enum SignerSerialPolicySpec {
     /// Use the same serial number as the unsigned zone.
     Keep,
 
     /// Increment the serial number on every change.
-    #[default]
     Counter,
```

Comment:
Is the lost default here and on `Nsec3OptOutPolicy` set somewhere else, or lost entirely?